src/Meshes/instancedMesh.ts

```diff
@@ -34,6 +34,7 @@ Mesh._instancedMeshFactory = (name: string, mesh: Mesh): InstancedMesh => {
 export class InstancedMesh extends AbstractMesh {
     private _sourceMesh: Mesh;
     private _currentLOD: Mesh;
+    private _billboardWorldMatrix = new Matrix();

```

Comment:
can we make it static please? So we don't create one per mesh

Reply:
We can't have it static because else all instances would use the same matrix.

Reply:
oh ok, I did not check the code corectly, I thought it was a temporary matrix used as an intermediary

In this case please simply delay creating it

Reply:
Done